src/env/dev/terraform.tfvars

```
@@ -12,7 +12,8 @@ tags = {
 cidr_vnet              = ["10.1.0.0/16"]
 cidr_subnet_appgateway = ["10.1.128.0/24"]
 ## ..
-cidr_subnet_mock_ec = ["10.1.240.0/29"]
+cidr_subnet_mock_ec  = ["10.1.240.0/29"]
+cidr_subnet_mock_psp = ["10.1.240.7/29"]
```

```suggestion
cidr_subnet_mock_psp = ["10.1.240.8/29"]
```
7 in 0/29 is the subnet broadcast address